packages/yourloops/lib/api.ts

```diff
@@ -276,6 +301,32 @@ class API extends EventTarget {
     return user;
   }
 
+  public async flagPatient(userId: string): Promise<string[]> {
```

Comment:
i think that it should be move in a dedicated api class

Reply:
Waiting for your api's PRs ;-)
For now just use the current code.